corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -42,7 +42,8 @@ const PricingTable = function (options) {
     self.isNextPlanDowngrade = self.nextSubscription && !self.isNextPlanPaused;
 
     self.oSelectedEdition = ko.observable(options.currentEdition);
-    self.oShowAnnualPricing = ko.observable(options.currentIsAnnualPlan);
+    self.oShowAnnualPricing = ko.observable(
+        self.currentIsAnnualPlan || self.isCurrentPlanFreeEdition || self.isCurrentPlanPaused);
```

Comment:
Not being too familiar with this code, it isn't easy to tell what all of the possible variables that _could_ factor into deciding whether or not to show annual pricing could be. If showing annual pricing is the default, I would almost imagine we would say something like "if the current plan is not X, show annual pricing". Not sure if that is possible and/or would even help, but just some late night thoughts 😅

Reply:
Agreed, but we'd still likely need logic like this somewhere along the line. The existing checks are all on the software plan for `is_annual_plan` -- which on Paused or Free plans is always false. There is no property `is_monthly_plan` because monthly is treated as default in backend code.

Would it be clearer to turn this logic into a variable, maybe `isCurrentPlanPaidMonthly` (naming is tough) and have `oShowAnnualPricing` use that as its default?